TUTORIAL.md

```diff
@@ -0,0 +1,384 @@
+# ZNAP Tutorial - **Donate/Pay a user**
+
+## Description
+
+Alice is a well-known influencer on Twitter. She usually shares research and valuable insights with her audience free of charge. She decides she wants to start a donation campaign for herself. She spins up a “Donate to me” Blink using the Actions Stack and shares the link to Sphere on Twitter.The link unfurls into an actionable Blink containing the following information:
+
+1. Her image, name, description
+2. 3 buttons: 1 SOL, 5 SOL, 10 SOL
+3. 4th button with an input field: “Enter a custom SOL amount”
+
+## Brief review
+
+Solana Actions are APIs that adhere to the Solana Actions Specification. Our Solana Actions must be capable of returning:
+
+1. Action Metadata
+2. A transaction to be sent to the Solana blockchain.
+
+## Let's Start!
+
+First, we will create our workspace using `znap-cli`. If you do not have `znap-cli` installed, follow our guide [here](https://github.com/heavy-duty/znap/blob/master/INSTALLATION.md).
+
+Next, let's create a new project with `znap init <project-name>`.
+
+### 1. Let's create our action collection for Alice's campaign
+
+In the root directory of your project, run the following command to create a collection: `znap new alice_campaign`. This command will create a new action collection called 'alice_campaign' inside the `/collections` folder, where we will place our actions for Alice.
```

Comment:
This command should be called with snake case `alice-campaign`. The name of the collection's folder also should be with snake case.

The exception here is the `pub mod alice_campaign` whish should be using the underscore notation.

Reply:
Done